packages/graphql/lib/src/core/raw_operation_data.dart

```diff
@@ -13,7 +13,8 @@ class RawOperationData {
     @required this.document,
     Map<String, dynamic> variables,
     String operationName,
-  })  : _operationName = operationName,
+  })  : assert(document != null),
```

Comment:
At the comment earlier we had a readable error. No we have an failing assertion on the `RawOperation` that might be a load harder to debug for someone new to the library.

Reply:
added a the message back in on failed assertion